packages/databricks-vscode/src/configuration/LoginWizard.ts

```diff
@@ -384,25 +385,36 @@ export async function saveNewProfile(
         throw new Error("Can't save empty auth provider to a profile");
     }
 
-    const {path: configFilePath} = await loadConfigFile(
-        workspaceConfigs.databrickscfgLocation
-    );
+    const configFilePath: string = getDatabricksConfigFilePath().fsPath;
+    let shouldBackup = true;
+    try {
+        await loadConfigFile(configFilePath);
```

Comment:
Is the bug here in that we are failing to save the profile if the cfg file doesn't exist?

Reply:
Yep